### PR TITLE
Exchange rate module

### PR DESCRIPTION
**Pull request by tobes**
Now that the exchange rate of my country is in free fall.

I thought an exchange rate module might be a fun way to watch the car crash.

### CODE REVIEW

**Comment by ultrabug** [comment]
Thanks for this *sad times* module, did you omit the standalone testing part on purpose ?

**Comment by tobes** [comment]
> did you omit the standalone testing part on purpose ?

yes because it needs https://github.com/ultrabug/py3status/pull/312

So if you pull this I'll add the test there, or here if that is pulled first.

I need to add a timeout and a few things to this PR still, I'll ping you when I've updated

**Comment by tobes** [comment]
Ok I've added the timeout/bad connection bits

**Comment by ultrabug** [comment]
Let's work on #312 first then.

File: py3status/modules/exchange_rate.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+View foreign exchange rates.
```
Comment (ultrabug) [inline py3status/modules/exchange_rate.py]: Display foreign exchange rates.

(more in line with other modules)

**Comment by tobes** [comment]
Test now added (and minor doc change you wanted)

**Comment by ultrabug** [comment]
:+1: thank you !

**Pull request merged**